src/components/screens/home/home-screen.tsx

```diff
@@ -1,7 +1,13 @@
+/**
+ * @module HomeScreen
+ * @date 9/15/2023 - 11:39:05 AM
+ *
+ * @returns {*}
```

Comment:
Remove unnecessary details and add the component descriptions e.g. HomeScreen Component